### ISSUE TITLE
Fuse Markdown doesn't support enhanced markdown syntax

### ISSUE THREAD

**Issue by reech-florian** [magidoc-org/magidoc#273]
### Describe the bug

Using the enhanced markdown syntax in the GraphQL schema break the build. According to the logs, this seems to come from Fuse markdown, which doesn't have an extractor for "notification" (in my case).

### Reproduction

GraphQL schema

```graphql
"The **dummy** enum"
enum DummyEnum {
  "A **value** of the enum."
  DummyTest

  """
  Another **value** of __the enum__.  

  :::notification type="warning"
  The notification that break the build
  :::
  """
  AnotherDummyTest
}
```

### Logs

```shell
Error: No extractor found for token type: notification
    at file:///tmp/carbon-multi-page@4.1.2/node_modules/.pnpm/@magidoc+plugin-fuse-markdown@4.1.2_fuse.js@6.6.2/node_modules/@magidoc/plugin-fuse-markdown/build/markdown/extract.js:70:23
    at Array.reduce (<anonymous>)
    at file:///tmp/carbon-multi-page@4.1.2/node_modules/.pnpm/@magidoc+plugin-fuse-markdown@4.1.2_fuse.js@6.6.2/node_modules/@magidoc/plugin-fuse-markdown/build/markdown/extract.js:67:23
    at file:///tmp/carbon-multi-page@4.1.2/node_modules/.pnpm/@magidoc+plugin-fuse-markdown@4.1.2_fuse.js@6.6.2/node_modules/@magidoc/plugin-fuse-markdown/build/markdown/extract.js:57:39
    at Array.forEach (<anonymous>)
    at extractTokens (file:///tmp/carbon-multi-page@4.1.2/node_modules/.pnpm/@magidoc+plugin-fuse-markdown@4.1.2_fuse.js@6.6.2/node_modules/@magidoc/plugin-fuse-markdown/build/markdown/extract.js:17:12)
    at extract (file:///tmp/carbon-multi-page@4.1.2/node_modules/.pnpm/@magidoc+plugin-fuse-markdown@4.1.2_fuse.js@6.6.2/node_modules/@magidoc/plugin-fuse-markdown/build/markdown/extract.js:7:12)
    at getDescription (file:///tmp/carbon-multi-page@4.1.2/node_modules/.pnpm/@magidoc+plugin-fuse-graphql@4.1.2_fuse.js@6.6.2/node_modules/@magidoc/plugin-fuse-graphql/build/indexer/graphql/description.js:8:12)
    at file:///tmp/carbon-multi-page@4.1.2/node_modules/.pnpm/@magidoc+plugin-fuse-graphql@4.1.2_fuse.js@6.6.2/node_modules/@magidoc/plugin-fuse-graphql/build/indexer/graphql/types.js:78:26
    at Array.map (<anonymous>)
```

### System Info

```shell
I run Magidoc in a Docker container that use the image `node:lts-alpine`. I have the latest version of Magidoc (4.1.2) and pnpm in.
```

### Severity

Annoyance

**Comment by pelletier197** [comment]
Hey! Sorry for the very late reply. I have just come to figure out your issue. I'll be releasing a fix very soon!

**Comment by pelletier197** [comment]
This should be fixed in 4.1.4. You may just re-open the issue if there's a problem! Thank you for raising that issue.

**Comment by reech-florian** [comment]
Thank you a lot !
